package.json

```diff
@@ -20,19 +20,18 @@
     "redux": "^4.0.1",
     "redux-thunk": "^2.3.0",
     "socket.io": "^2.2.0",
-    "socket.io-client": "^2.2.0",
     "socket.io-mock": "^1.2.3"
   },
   "scripts": {
     "start": "node server.js",
-    "start:devserver": "npx cross-env NODE_ENV=testing nodemon server.js --watch ./server --watch ./server.js",
+    "start:devserver": "concurrently \"nodemon ./server/proxy/server.js\" \"npx cross-env NODE_ENV=testing nodemon --inspect server.js --watch ./server --watch ./server.js\"",
     "start:devclient": "react-scripts start",
     "build": "react-scripts build",
     "test:client": "react-scripts test --coverage --watchAll=false",
     "eject": "react-scripts eject",
     "cy:run": "cypress run",
     "cypress:open": "cypress open",
-    "test:e2e": "concurrently --success first --kill-others \"npx cross-env NODE_ENV=testing node server.js\" \"node run-cypress-test.js -r tap\""
+    "test:e2e": "concurrently --success first --kill-others \"nodemon ./server/proxy/server.js\" \"npx cross-env NODE_ENV=testing node server.js\" \"node run-cypress-test.js -r tap\""
```

Comment:
Probably don't need to used `nodemon` here, you don't need the restart functionailty do you? Also, why `npx`? Should we not just add the package we're using to the dev dependencies and use `npm`?

Reply:
That's a good point, I didn't pay attention to this.. I'll change it now

Reply:
> Should we not just add the package we're using to the dev dependencies and use npm?

that's more steps, right? and it relies on node_modules' being pathed correctly.

Reply:
`npx` is for the `cross-env` thing, it's needed for adding the environment variables that we want apparently, Alex and I found it

Reply:
OK, maybe there's more to `npx` than I thought.